### DISCUSSION TITLE
Nuget update to Math.NET Spatial v0.7.0

### DISCUSSION THREAD

**Discussion by t0bde** [mathnet/mathnet-spatial#243] [General]
Hi guys, first of all I would like to say that I love your library! Would it be possible to make the Math.NET Spatial v0.7.0 version available on Nuget?

**Comment by jkalias** [comment]
I would certainly love to, but even though I am a co-maintainer of this repo, I don't have publish rights to Nuget. Perhaps @JohanLarsson could try instead? @cdrnet is relatively quiet, also in the MathNet.Numerics repo, so I don't know how to interpret this.

I don't know if this helps, maybe you can try https://github.com/jkalias/mathnet-spatial/releases?

**Reply by t0bde** [reply]
Thank you for the quick reply. Unfortunately, we use Nuget for package management in our projects (version check, target platform and so on). Unfortunately, we do not include “loose” assemblies. But that's not a big problem. I only use one method from version 0.7, so I'll build a workaround. Thanks again.

**Comment by SkinnySackboy** [comment]
This would be great.
